scanner/src/main/kotlin/Main.kt

```diff
@@ -213,106 +213,118 @@ object Main {
 
         println("Using scanner '$scanner'.")
 
+        val scanSummary: ScanSummary = dependenciesFile?.let {
+            scanDependenciesFile(it)
+        } ?: inputPath!!.let {
```

Comment:
This can now actually be simplified to `scanInputPath(inputPath!!)`.

Reply:
Done.